pgo/cmd/backrest.go

```diff
@@ -85,15 +87,49 @@ func showBackrest(args []string, ns string) {
 		for _, backup := range response.Items {
 			printBackrest(&backup)
 		}
-
 	}
-
 }
 
 // printBackrest
 func printBackrest(result *msgs.ShowBackrestDetail) {
 	fmt.Printf("%s%s\n", "", "")
-	fmt.Printf("%s%s\n", "", "backrest : "+result.Name)
-	fmt.Printf("%s%s\n", "", result.Info)
-
+	fmt.Printf("cluster: %s\n", result.Name)
+	fmt.Printf("storage type: %s\n\n", result.StorageType)
+
+	for _, info := range result.Info {
+		fmt.Printf("stanza: %s\n", info.Name)
+		fmt.Printf("    status: %s\n", info.Status.Message)
+		fmt.Printf("    cipher: %s\n\n", info.Cipher)
+
+		for _, archive := range info.Archives {
+			// this is the quick way of getting the name...alternatively we could look
+			// it up by ID
+			fmt.Printf("    %s (current)\n", info.Name)
+			fmt.Printf("        wal archive min/max (%s)\n\n", archive.ID)
+
+			// iterate trhough the the backups and list out all the information
+			for _, backup := range info.Backups {
+				databaseSize, databaseUnit := getSizeAndUnit(backup.Info.Size)
+				databaseBackupSize, databaseBackupUnit := getSizeAndUnit(backup.Info.Delta)
+				repositorySize, repositoryUnit := getSizeAndUnit(backup.Info.Repository.Size)
+				repositoryBackupSize, repositoryBackupUnit := getSizeAndUnit(backup.Info.Repository.Delta)
+
+				// this matches the output format of pgbackrest info
+				fmt.Printf("        %s backup: %s\n", backup.Type, backup.Label)
+				fmt.Printf("            timestamp start/stop: %s / %s\n",
```

Comment:
Is there a reason not to use `\t` here? Assuming it is to keep it consistent with what was already being output to ensure non-breakage of consumers?

Reply:
@abrightwell I was matching the exact output from `pgbackrest info`, so in case anyone is presently parsing it, it does not break it.